react/src/pages/datasets/DatasetTable.tsx

```diff
@@ -0,0 +1,147 @@
+    participant_codename: string;
+    family_codename: string;
+    dataset_type: string;
+    input_hpf_path?: string;
+    notes?: string;
+    condition: string;
+    created: Date;
+    created_by: string;
+    updated: Date;
+    updated_by: string;
+}
+
+const useStyles = makeStyles(theme => ({
+    chip: {
+        color: "primary",
+        marginRight: '10px',
+        colorPrimary: theme.palette.primary,
+    },
+    chipBar: {
+        marginLeft: '24px',
+        marginTop: '6px'
+    }
+}));
+
+function createDataset(
+    dataset_id: number,
+    tissue_sample_type: string,
+    participant_codename: string,
+    family_codename: string,
+    dataset_type: string,
+    condition: string,
+    created: Date,
+    created_by: string,
+    updated: Date,
+    updated_by: string) {
+    return {
+        dataset_id, tissue_sample_type, participant_codename, family_codename,
+        dataset_type, condition, created, created_by, updated, updated_by
+    };
+}
+
+const rows: Dataset[] = [
+    createDataset(1, 'Blood', 'AA0001', '3001', 'WGS', 'Somatic', new Date(), 'CHEO', new Date(), 'CHEO'),
+    createDataset(2, 'Blood', 'AA0002', '3002', 'WGS', 'Somatic', new Date('2020-09-01'), 'CHEO', new Date('2020-09-01'), 'CHEO'),
+    createDataset(3, 'Blood', 'AA0003', '3003', 'WGS', 'Somatic', new Date('2020-09-01'), 'CHEO', new Date('2020-09-01'), 'CHEO'),
+    createDataset(4, 'Skin', 'BB0001', '2001', 'WES', 'Somatic', new Date('2020-09-01'), 'CHEO', new Date('2020-09-01'), 'CHEO'),
+    createDataset(5, 'Blood', 'BB0002', '2002', 'WGS', 'Somatic', new Date('2020-09-01'), 'CHEO', new Date('2020-09-01'), 'CHEO'),
+    createDataset(6, 'Saliva', 'BB0003', '2003', 'WES', 'Somatic', new Date('2020-08-22'), 'ACH', new Date(), 'ACH'),
+    createDataset(7, 'Blood', 'AA0004', '3012', 'WGS', 'Somatic', new Date('2020-08-22'), 'ACH', new Date(), 'ACH'),
+    createDataset(8, 'Skin', 'AA0005', '3013', 'WES', 'Somatic', new Date('2020-08-22'), 'ACH', new Date(), 'ACH'),
+];
+
+export default function DatasetTable() {
+    const classes = useStyles();
+    const [showRunner, setRunner] = useState(false);
+    const [activeParticipants, setActiveParticipants] = useState<Dataset[]>([]);
+    const [datasetType, setDatasetType] = useState("");
+
+    return (
+        <div>
+            <AnalysisRunnerDialog
+                participants={activeParticipants}
+                open={showRunner}
+                onClose={() => setRunner(false)}
+            />
+            <MaterialTable
+                columns={[
+                    { title: 'Participant', field: 'participant_codename', editable: 'never' },
+                    { title: 'Family', field: 'family_codename', editable: 'never' },
+                    { title: 'Tissue sample', field: 'tissue_sample_type' },
+                    { title: 'Dataset type', field: 'dataset_type', defaultFilter: datasetType },
+                    { title: 'Condition', field: 'condition' },
```

Comment:
These 3 columns display values chosen from a pre-defined list (eg. Tissue Sample: Blood, Saliva, Skin, etc.), and so you should edit the column definitions to reflect that. 

MaterialTable has an option for columns called `lookup` to provide said values to select from when filtering or editing rows.

eg. `{ title: 'Condition', field: 'condition', lookup: { 0: 'Control', 1: 'GermLine', 2: 'Somatic' } },` I think the key determines what order they are displayed in, but it otherwise doesn't matter.

Reply:
The key is the actual value in the table, and the value is displayed in the filter dropdown. If numeric keys are used then the cells for this column are blank, I think because they don't match.